gmail/tsconfig.json

```diff
@@ -2,7 +2,6 @@
   "compilerOptions": {
     "outDir": "./build",
     "baseUrl": ".",
-    "sourceMap": true,
```

Comment:
can't we keep that ? (useful to debug when we have error message)

Reply:
Does it actually work on App Scripts though? It seemed like it only gave errors inside the `.gs` files and just ignored the sourcemap.

I'll enable it in rollup for now either way, it does provide source mapping to the original files (unrolled) files :)